src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -335,14 +335,14 @@ export const makeDatasetTableHeader = (datasets: DatasetTerm[], selected: number
       })
     },
     {
-      label: 'Donor Size',
+      label: 'Participant Count',
       sortable: true,
-      cellStyle: makeHeaderStyle(cellWidths.donorSize),
+      cellStyle: makeHeaderStyle(cellWidths.participantCount),
       cellDataFn: (dataset: DatasetTerm) => {
         const donorSize = isNaN(dataset.participantCount) ? '' : dataset.participantCount.toString();
```

Comment:
Do you still want to call this donorSize?  I see another reference to donorSize further down in the code.

Reply:
Thank you, missed that one